geometry_msgs/Twist user_input
geometry_msgs/Twist ca_twist
geometry_msgs/Pose ee_pose
---
geometry_msgs/Twist assisted_twist
float32[] distr_prob
int8 index_max
